examples/artifacts/plot_correct_ajdc_EEG.py

```diff
@@ -0,0 +1,179 @@
+"""
+===============================================================================
+Artifact Correction by AJDC-based Blind Source Separation
+===============================================================================
+
+Blind source separation (BSS) based on approximate joint diagonalization of
+Fourier cospectra (AJDC), applied to artifact correction of EEG [1].
+"""
+# Authors: Quentin Barthélemy & David Ojeda.
+# EEG signal kindly shared by Marco Congedo.
+#
+# License: BSD (3-clause)
+
+import numpy as np
+from scipy.signal import welch
+from matplotlib import pyplot as plt
+
+from mne import create_info
+from mne.io import RawArray
+from mne.viz import plot_topomap
+from mne.preprocessing import ICA
+
+from pyriemann.spatialfilters import AJDC
+from pyriemann.utils.viz import plot_cospectra
+
+
+###############################################################################
+
+
+def read_header(fname):
+    """Read the header of sample-blinks.txt"""
+    with open(fname, "r") as f:
+        content = f.readline().split()
+        return content[:-1], int(content[-1])
+
+
+###############################################################################
+# Load EEG data
+# -------------
+
+fname = '../data/sample-blinks.txt'
```

Comment:
pypi limits source distributions and it's a good idea to keep the soruce repo as small as possible.
Can you avoid using a .txt file for this reason? Maybe .npy or .txt.gz ? numpy will have no
problem with .txt.gz